android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -107,6 +107,7 @@ public void run() {
       emptyGroupIdQueue();
       mSecondScanComplete = true;
       mSwipeRefreshWidget.setRefreshing(false);
+      mScanningAnimationTextView.setAlpha(0f);
```

Comment:
I see what you mean, it only has any visible effect in this case since in all other cases the empty view is hidden.  It's fine with me to leave it as-is since you'll be replacing it soon with a better "no results" empty view.